### PR TITLE
Fix for updated website

### PR DESCRIPTION
**Pull request by meriko**
Dependent on https://github.com/plexinc-plugins/Services.bundle/pull/812

### CODE REVIEW

**Comment by meriko** [comment]
https://forums.plex.tv/index.php/topic/162787-hgtvca-channel-content-unavailable/

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -1,245 +1,140 @@
...
-	content = JSON.ObjectFromURL(FEED_LIST % (network[0], network[1]))
-
-	for item in content['items']:
-		if WantedCats(item['parent'],cats):
-			title = item['fullTitle'].split('/')[2]
-			iid = item['ID']
-			thumb_url = item['thumbnailURL']
-			
-			# a couple of shows have "Full Episodes" instead of "Season [0-9]"
-			if RE_SEASON_TEST.search(item['fullTitle']) or "Full Episodes" in item['fullTitle']:
```
Comment (Gerk) [inline Contents/Code/__init__.py]: Have they really cleaned up all the old metadata in their older feeds?  Is this sort of stuff no longer needed?  In the past there were a LOT of corner cases in order to get all the shows and seasons to show up ...

**Comment by meriko** [comment]
The channel now requests all videos for all shows(and seasons) and only adds the shows that has full episodes available.

But, it doesn't seem that all shows are present in the new feed? An example is "Best of Holmes on Homes" ....

**Comment by Gerk** [comment]
The older (current) version of the channel didn't scrape the site at all, it used backend show title listings (FEED_LIST?) and then grabbed individual listings of each show (FEEDS_LIST?) -- it's been a while but I know that there were definitely two different feeds, one for shows one for episodes.

Also it used to differentiate between full episodes and clips (I personally hate having them all piled up together, when I want to watch full shows I don't care about clips and vice-versa).  

Not saying this is a requirement but it's a nice thing to have.  I haven't got time to test or work on this personally so I guess I can't complain, glad you have things at least working ... but I know that at least my wife is gonna freak out if shows are missing and/or you can't get at full episodes and clips separately -- which means I'll have to come back and work on it at some point LOL.

**Comment by Gerk** [comment]
Also it's possible that they axed some of the older shows (like best of holmes on holmes), who knows what they are doing over there.

**Comment by meriko** [comment]
In this new version, clips and full episodes are still separated. For each show there is a Clips folder at the top followed by all full episodes. I made it like that so you don't have to navigate one extra level if you want full episodes(e.g. avoid a choice navigation).

Since I don't personally use this channel, you're absolutely free to update according to your wife's specification :)

Also, the new version also uses an API, i.e. doesn't scrape the site. But it's strange that not all shows are present in this feed? Maybe they are using multiple feeds, but the only feed url I've seen so far is the one used in the channel.

I'll try to log/sniff some more network traffic ...

**Comment by meriko** [comment]
I've now found an additional feed, will reopen this later

**Comment by meriko** [comment]
I think this is now dependent on https://github.com/plexinc-plugins/Services.bundle/pull/814

**Pull request merged**